pig/src/org/partiql/pig/domain/parser/ParserErrorContext.kt

```diff
@@ -42,6 +39,9 @@ sealed class ParserErrorContext(val msgFormatter: () -> String): ErrorContext {
         override fun hashCode(): Int = 0
     }
 
+    data class CouldNotFindImportedTypeUniverse(val tag: String)
+        : ParserErrorContext({ "Could not find imported type universe: $tag" })
```

Comment:
nit: could change usages of `import` to `include`
```suggestion
    data class CouldNotFindIncludedTypeUniverse(val tag: String)
        : ParserErrorContext({ "Could not find included type universe: $tag" })
```

Reply:
:heavy_check_mark: